surrealdb/common/exceptions.py

```diff
@@ -17,8 +17,8 @@
 
 
 class SurrealException(Exception):
-    pass
+    """Exception raised for errors with the SurrealDB http client."""
```

Comment:
"http client" is wrong, this is just a base class for any errors related to surreal